src/app/app.component.html

```diff
@@ -87,6 +87,6 @@
     -->
   </ul>
 </div>
-<main class="container-fluid container-pf-nav-pf-vertical">
+<main class="container-fluid container-pf-nav-pf-vertical" *ngIf="loggedIn">
```

Comment:
Not sure this would be a good way to do this in the future, but is fine for now. We can eventually include different templates depending on whether or not the user is logged.